packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -399,10 +426,14 @@ class CloudFormation {
         });
     } catch (error) {
       this.progressBar?.stop();
-      throw new AmplifyFault('ResourceNotReadyFault', {
-        message: error.message,
-        code: error.code,
-      }, error);
+      throw new AmplifyFault(
```

Comment:
Do we need to add error.detail here as well or are we fetching it from downstream  exception ?